docs/source/icml.py

```diff
@@ -0,0 +1,556 @@
+'''
```

Comment:
Are we using this file?

Reply:
I see that the python notebook is importing the file but we are not using it, right?

Reply:
You are right, we are not using it. It was placed so that we could execute the pynb on the fly. But, later it was found that we do not need to execute the pynb if it already has the responses. I'll remove the file. Thanks Melisa.